app/src/main/java/org/gdg/frisbee/android/widget/FeedbackFragment.java

```diff
@@ -103,6 +105,14 @@ public void onCreate(Bundle savedInstanceState) {
         addProperty("loggedIn", PrefUtils.isSignedIn(getActivity())); // Optionally add some properties
         addProperty("appStarts", PrefUtils.getAppStarts(getActivity()));
         buildProperties();
+
+        setRetainInstance(true);
```

Comment:
I didn't know that there is error Callback method. We can do that. It would
make sense. Btw the dialog does not survive orientation change.

On Tue, Jan 19, 2016, 11:19 Friedger Müffke <notifications@github.com>
wrote:

> In app/src/main/java/org/gdg/frisbee/android/widget/FeedbackFragment.java
> <https://github.com/gdg-x/frisbee/pull/536#discussion_r50088243>:
>
> > @@ -103,6 +105,14 @@ public void onCreate(Bundle savedInstanceState) {
> >          addProperty("loggedIn", PrefUtils.isSignedIn(getActivity())); // Optionally add some properties
> >          addProperty("appStarts", PrefUtils.getAppStarts(getActivity()));
> >          buildProperties();
> > +
> > +        setRetainInstance(true);
>
> What about leaving the dialog open until doorbell comes back with a result?
>
> There is also the setErrorCallback
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/536/files#r50088243>.
>